app/src/main/java/ar/com/wolox/android/features/homepage/news/NewsViewHolder.kt

```diff
@@ -19,23 +16,15 @@ class NewsViewHolder(view: View) : RecyclerView.ViewHolder(view) {
     val viewTime: TextView = view.findViewById(R.id.timeTextView)
     val viewImage: ImageView = view.findViewById(R.id.newsImageView)
 
-    fun populate(dataSet: News?, context: Context) {
+    fun populate(dataSet: NewFromPage?, clickListener: OnItemClickListener, userId: Int) {
         with(dataSet!!) {
             viewTitle.text = commenter
             viewDescription.text = comment
-            viewTime.text = deltaTime(updated_at)
+            viewTime.text = deltaTime(created_at)
```

Comment:
I don't like having a function declared in a random file. I suggest to move this function to a String extension function. What do you think?